src/Developer/OilPaintingImageFilter.cxx

```diff
@@ -0,0 +1,58 @@
+#include "itkImage.h"
```

Comment:
I see source code is moved here, but no CMake or RST. Is that a problem?

Reply:
I don't know.  I could not find how this code was used on the media wiiki WikiExamples, so I just ported it without knowing how/if it will be used.

Some of the items in the "Developer" directory are used to support building of other MediaWiki content pages.